hooks/ios/iOSCopyPreferences.js

```diff
@@ -2,39 +2,43 @@ const et = require('elementtree');
 const path = require('path');
 const fs = require('fs');
 const plist = require('plist');
-const child_process = require('child_process');
 const { ConfigParser } = require('cordova-common');
-const { Console } = require('console');
 
 module.exports = function (context) {
 
     const ServiceEnum = Object.freeze({"ApplePay":"1", "GooglePay":"2"})
-    var projectRoot = context.opts.cordova.project ? context.opts.cordova.project.root : context.opts.projectRoot;
-
-    var merchant_id = "";
-    var merchant_name = "";
-    var merchant_country_code = "";
-    var payment_allowed_networks = [];
-    var payment_supported_capabilities = [];
-    var payment_supported_card_countries = [];
-    var shipping_supported_contacts = [];
-    var billing_supported_contacts = [];
-    var payment_gateway = "";
-    var payment_request_url = "";
-    var stripe_publishable_key = "";
-
-    var appNamePath = path.join(projectRoot, 'config.xml');
-    var appNameParser = new ConfigParser(appNamePath);
-    var appName = appNameParser.name();
-
+    let projectRoot = context.opts.cordova.project ? context.opts.cordova.project.root : context.opts.projectRoot;
+
+    let merchant_id = "";
+    let merchant_name = "";
+    let merchant_country_code = "";
+    let payment_allowed_networks = [];
+    let payment_supported_capabilities = [];
+    let payment_supported_card_countries = [];
+    let shipping_supported_contacts = [];
+    let billing_supported_contacts = [];
+    let payment_gateway = "";
+    let payment_request_url = "";
+    let stripe_publishable_key = "";
+
+    let appNamePath = path.join(projectRoot, 'config.xml');
+    let appNameParser = new ConfigParser(appNamePath);
+    let appName = appNameParser.name();
+
+    let wwwFolder = "platforms/ios/www";    
```

Comment:
From the looks of it, this is only used once (line 32) and `platformPath` is reused many times so maybe we could remove this and replace is usage with `platformPath + "/www"'?